src/style.css

```diff
@@ -1 +1,81 @@
 /* add styles here */
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
```

Comment:
The `font-weight` property should be specified using numeric values, not pixels. For example, `font-weight: 300;`.